rebar.config

```diff
@@ -1,7 +1,7 @@
 {erl_opts, [nowarn_unused_type, debug_info, {d, maps_support}]}.
 {deps, [{hackney, "1.18.0"},
         {jsx, "3.0.0"},
-        {aws_signature, "0.3.3"}
+        {aws_beam_core, "1.0.0"}
```

Comment:
Not important for this PR: Since you are using hex.pm, you can use [semver-like syntax](https://rebar3.org/docs/configuration/dependencies/)

Reply:
I'm a 🐔 when it comes to the semver-like syntax. Even though `aws_beam_core` is owned by us, I'd prefer hard-pinning it just to ensure that updated dependencies are explicitly bumped rather than implicitly.